build-test.xml

```diff
@@ -9980,8 +9980,20 @@ org.apache.catalina.tribes.level = FINE]]>
 	</target>
 
 	<target name="deploy-maven-module">
+		<if>
+			<not>
+				<isset property="build.profiles" />
+			</not>
+			<then>
+				<var name="build.profiles" value="" />
+			</then>
+			<else>
+				<var name="build.profiles" value="-P ${build.profiles}" />
```

Comment:
@Kvale1733 if ${build.profiles} is unSet, then does it default to *${build.profiles}* like it would in Poshi? If so then this is fine, otherwise the then branch is un-needed

Reply:
Yup If I remove it then it actually outputs ${buld.profiles} which breaks the command